src/fprime_gds/executables/cli.py

```diff
@@ -185,29 +188,85 @@ def handle_arguments(self, args, **kwargs):
         Returns: namespace with processed results of arguments.
         """
 
-    @staticmethod
+    @classmethod
+    def parse_known_args(
+        cls,
+        parser_classes,
+        description="No tool description provided",
+        arguments=None,
+        **kwargs,
+    ):
+        """Parse and post-process arguments
+
+        Create a parser for the given application using the description provided. This will then add all specified
+        ParserBase subclasses' get_parser output as parent parses for the created parser. Then all of the handle
+        arguments methods will be called, and the final namespace will be returned. This will allow unknown arguments
+        which are returned as the last tuple result.
+
+        Args:
+            parser_classes: a list of ParserBase subclasses that will be used to
+            description: description passed ot the argument parser
+            arguments: arguments to process, None to use command line input
+        Returns: namespace with all parsed arguments from all provided ParserBase subclasses
+        """
+        return cls._parse_args(parser_classes, description, arguments, allow_unkowns=True, **kwargs)
```

Comment:
This function is called `parse_known_args` yet uses `allow_unkowns=True` which is a little confusing. Should the keyword name be refined?